core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/bundle/BundleRequests.java

```diff
@@ -18,27 +18,25 @@
 /**
  * @since 8.0
  */
-public final class BundleRequests {
+public class BundleRequests {
```

Comment:
You can keep the final keyword, classes like this are not meant to be subclassed.